packages/web/util.js

```diff
@@ -520,3 +520,23 @@ export function clearLog() {
 
     hideValidationStatusTable()
 }
+
+/**
+ * Save current TD/TM as a compressed string in URL fragment.
+ * @param {string} docType "td" or "tm"
+ */
+export async function save(docType) {
```

Comment:
this needs to be smarter to take tm and tds into account, somehow. Currently, if a TM is saved, the TD window opens but has nothing in it. So ideally, if a TM is saved and opened, the TM windows should open first.
It would be also cool to think of a way to save both the TM and TD at the same time. Maybe for two birds one stone, something like query parameters can be used.

Reply:
Done